pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceBaseArguments.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.testclient;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import com.beust.jcommander.Parameter;
+import java.io.FileInputStream;
+import java.util.Properties;
+import lombok.SneakyThrows;
+
+
+public abstract class PerformanceBaseArguments {
+
+    @Parameter(names = { "-h", "--help" }, description = "Help message", help = true)
+    boolean help;
+
+    @Parameter(names = { "-cf", "--conf-file" }, description = "Configuration file")
+    public String confFile;
+
+    @Parameter(names = { "-u", "--service-url" }, description = "Pulsar Service URL")
+    public String serviceURL;
+
+    @Parameter(names = { "--auth-plugin" }, description = "Authentication plugin class name")
+    public String authPluginClassName;
+
+    @Parameter(
+            names = { "--auth-params" },
+            description = "Authentication parameters, whose format is determined by the implementation "
+                    + "of method `configure` in authentication plugin class, for example \"key1:val1,key2:val2\" "
+                    + "or \"{\"key1\":\"val1\",\"key2\":\"val2\"}.")
+    public String authParams;
+
+    @Parameter(names = {
+            "--trust-cert-file" }, description = "Path for the trusted TLS certificate file")
+    public String tlsTrustCertsFilePath = "";
+
+    @Parameter(names = {
+            "--tls-allow-insecure" }, description = "Allow insecure TLS connection")
+    public Boolean tlsAllowInsecureConnection = null;
+
+    @Parameter(names = {
+            "--tls-hostname-verification" }, description = "Enable TLS hostname verification")
```

Comment:
For other CLI tools in the project, we already use either `--tls-enable-hostname-verification` or `--hostname-verification-enabled`. Since this hasn't been released yet, I'd prefer not to add another name for the same configuration. I like `--hostname-verification-enabled`, do you have a preference @nicoloboschi?

https://github.com/apache/pulsar/blob/09be79742473b7686f44d3f47e7c10fb9b441792/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java#L73

https://github.com/apache/pulsar/blob/09be79742473b7686f44d3f47e7c10fb9b441792/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java#L170

Reply:
Good point @michaeljmarshall
Since other TLS params in the perf cli have `--tls` as prefix, I'd prefer to go with `--tls-enable-hostname-verification`

Reply:
Good point, I agree with using `--tls-enable-hostname-verification`.